lib/types/api.types.ts

```diff
@@ -22,6 +22,26 @@ import { EntryFieldInfo, FieldInfo } from './field.types'
 import { Adapter, KeyValueMap } from 'contentful-management/types'
 
 /* User API */
+export declare const ACCEPT = 'ACCEPT'
+export declare const DENY = 'DENY'
+declare type ConsentRecordKey =
+  | 'ANALYTICS'
+  | 'ESSENTIAL'
+  | 'MARKETING'
+  | 'PERSONALIZATION'
+  | 'OPT_OUT'
+  | 'STORAGE'
+
+declare type ConsentData = {
+  uuid: string
+  expirationDate: string | null
+  consentRecord: Record<ConsentRecordKey, 'ACCEPT' | 'DENY'>
+  rawConsentRecord: string
```

Comment:
Why expose `uuid` and `rawConsentRecord`? I think that will cause confusion for the users.